Spatial_Engine/Query/ControlPoints.cs

```diff
@@ -30,6 +30,19 @@ namespace BH.Engine.Spatial
 {
     public static partial class Query
     {
+        /******************************************/
+        /****            IElement0D            ****/
+        /******************************************/
+
+        [Description("Queries the control points of the geometrical representation of the IElement0D. Always returns the point location due to zero-dimensionality of an IElement0D.")]
+        [Input("element0D", "The IElement0D with the geometry to get the control points from.")]
+        [Output("cPoints", "The control points of the geometrical representation of an IElement0D.")]
```

Comment:
```suggestion
        [Description("Queries the control points of the geometrical representation of an IElement0D. Always returns the point location due to zero-dimensionality of an IElement0D.")]
        [Input("element0D", "The IElement0D with the geometry to get the control points from.")]
        [Output("cPoints", "The control points of the geometrical representation of an IElement0D.")]
```
Changing the article to match the general pattern. (Also I think both are correct, but I like "an" better in these cases. Maybe a native speaker should decide).